iothub_client/tests/iothubtransport_mqtt_common_int_ut/iothubtransport_mqtt_common_int_ut.c

```diff
@@ -0,0 +1,1169 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Could you add a comment in the code here to clarify why we have a second unit test module for iothubtransport_mqtt_common?